source/datum.psd1

```diff
@@ -1,119 +1,46 @@
     Author            = 'Gael Colas'
 
-    # Company or vendor of this module
     CompanyName       = 'SynEdgy Limited'
 
-    # Copyright statement for this module
     Copyright         = '(c) 2020 Gael Colas. All rights reserved.'
 
-    # Description of the functionality provided by this module
     Description       = 'Module to manage Hierarchical Configuration Data.'
 
-    # Minimum version of the Windows PowerShell engine required by this module
-    # PowerShellVersion = ''
-
-    # Name of the Windows PowerShell host required by this module
-    # PowerShellHostName = ''
-
-    # Minimum version of the Windows PowerShell host required by this module
-    # PowerShellHostVersion = ''
-
-    # Minimum version of Microsoft .NET Framework required by this module
-    # DotNetFrameworkVersion = ''
-
-    # Minimum version of the common language runtime (CLR) required by this module
-    # CLRVersion = ''
-
-    # Processor architecture (None, X86, Amd64) required by this module
-    # ProcessorArchitecture = ''
-
-    # Modules that must be imported into the global environment prior to importing this module
     RequiredModules   = @(
         'powershell-yaml'
     )
 
-    # Assemblies that must be loaded prior to importing this module
-    # RequiredAssemblies = @()
-
-    # Script files (.ps1) that are run in the caller's environment prior to importing this module.
     ScriptsToProcess  = @(
         './ScriptsToProcess/Resolve-NodeProperty.ps1'
     )
 
-    # Type files (.ps1xml) to be loaded when importing this module
-    # TypesToProcess = @()
-
-    # Format files (.ps1xml) to be loaded when importing this module
-    # FormatsToProcess = @()
-
-    # Modules to import as nested modules of the module specified in RootModule/ModuleToProcess
-    # NestedModules = @()
-
-    # Functions to export from this module
     FunctionsToExport = ''
 
-    # Cmdlets to export from this module
-    #CmdletsToExport = '*'
-
-    # Variables to export from this module
-    #VariablesToExport = '*'
-
-    # Aliases to export from this module
     AliasesToExport   = ''
 
-    # List of all modules packaged with this module
-    # ModuleList = @()
-
-    # List of all files packaged with this module
-    # FileList = @()
-
-    # Private data to pass to the module specified in RootModule/ModuleToProcess
     PrivateData       = @{
 
         PSData = @{
 
-            # Tags applied to this module. These help with module discovery in online galleries.
             Tags         = @('Datum', 'Hiera', 'DSC', 'DesiredStateConfiguration', 'hierarchical', 'ConfigurationData', 'ConfigData')
 
-            # A URL to the license for this module.
             LicenseUri   = 'https://github.com/gaelcolas/Datum/blob/master/LICENSE'
 
-            # A URL to the main website for this project.
             ProjectUri   = 'https://github.com/gaelcolas/Datum/'
 
-            # A URL to an icon representing this module.
-            # IconUri = ''
```

Comment:
would be good to readd this at some point.

Reply:
Shall we use the DSC Community one from for example [ComputerManagementDsc.psd1](https://github.com/dsccommunity/ComputerManagementDsc/blob/master/source/ComputerManagementDsc.psd1)?

![dsccommunity](https://dsccommunity.org/images/DSC_Logo_300p.png)